CONTRIBUTING_CODE.md

```diff
@@ -1,51 +1,51 @@
 # Contributing to the Calico Codebase
 
+Thanks for considering contributing to Calico! This document outlines the canonical procedure for contributing new features
+and bugfixes to Calico - following these steps will help ensure that your contribution gets merged quickly and
+efficiently.
+
 ## Overview
 
-Features or any changes to the codebase should be done as follows:
-
-1.  Pull latest code in the **master** branch and create a feature
-    branch off this.
-1.  Implement your feature. Commits are cheap in Git; try to split up
-    your code into many. It makes reviewing easier as well as for
-    saner merging.
-    -   If your commit fixes an existing issue \#123, include the text
-        "fixes \#123" in at least one of your commit messages. This
-        ensures the pull request is attached to the existing issue (see
-        [How do you attach a new pull request to an existing issue on
-        GitHub?](https://stackoverflow.com/questions/4528869/how-do-you-attach-a-new-pull-request-to-an-existing-issue-on-github)).
-
-1.  Push your feature branch to GitHub. Note that before we can accept
-    your changes, you need to agree to one of our
-    contributor agreements. See [below](#contributor-agreements).
-1.  Create a pull request using GitHub, from your branch to master.
-1.  Reviewer process:
-    -   Receive notice of review by GitHub email, GitHub notification,
-        or by checking your assigned issues.
-    -   Make markups as comments on the pull request (either line
-        comments or top-level comments).
-    -   Make a top-level comment saying something along the lines of
-        “Fine; some minor comments” or “Some issues to address
-        before merging”.
-    -   If there are no issues, merge the pull request and close the
-        branch. Otherwise, assign the pull request to the developer and
-        leave this to them.
-
-1.  Developer process:
-    -   Await review.
-    -   Address code review issues on your feature branch.
-    -   Push your changes to the feature branch on GitHub. This
-        automatically updates the pull request.
-    -   If necessary, make a top-level comment along the lines of
-        “Please re-review”, assign back to the reviewer, and repeat
-        the above.
-    -   If no further review is necessary and you have the necessary
-        privileges, merge the pull request and close the branch.
-        Otherwise, make a top-level comment and assign back to the
-        reviewer as above.
+### Is this a new feature?
+
+If you'd like to add a new feature to Calico, please first open an issue describing the desired functionality. A Calico
+maintainer will work with you to come up with the correct design for the new feature. Once you've agreed a design, you can then
+start contributing code to the affect Calico repositories using the development process described below.
```

Comment:
definitely: "affect" --> "affected" 

But really, I wouldn't use "affected" here. I don't think it's quite the right word....I would go with "relevant" instead.